test/e2e/tls_test.go

```diff
@@ -115,43 +117,40 @@ func (data *TestData) checkTLS(t *testing.T, podName string, containerName strin
 	}
 
 	// 2. Set TLSMaxVersion to TLS1.2, then TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 should be used
-	stdouts = data.opensslConnect(t, podName, containerName, true, apiserver)
+	stdouts = data.curlTestTLS(t, podName, containerName, true, apiserver, dstIPv4, dstIPv6)
 	for _, stdout := range stdouts {
-		assert.True(t, strings.Contains(stdout, fmt.Sprintf("New, TLSv1.2, Cipher is %s", cipherSuiteStr)),
+		assert.True(t, strings.Contains(stdout, fmt.Sprintf("SSL connection using TLSv1.2 / %s", cipherSuiteStr)),
 			"Cipher Suite used by %s apiserver should be the TLS1.2 one '%s', output: %s", apiserverStr, cipherSuiteStr, stdout)
 	}
 }
 
-func (data *TestData) opensslConnect(t *testing.T, pod string, container string, tls12 bool, port int) []string {
+func (data *TestData) curlTestTLS(t *testing.T, pod string, container string, tls12 bool, port int, dstIPv4, dstIPv6 string) []string {
 	var stdouts []string
-	opensslConnectCommands := []struct {
-		enabled bool
-		ip      string
-		option  string
+	curlTLSCommands := []struct {
+		ip string
 	}{
 		{
-			clusterInfo.podV4NetworkCIDR != "",
-			"127.0.0.1",
-			"-4",
+			dstIPv4,
 		},
 		{
-			clusterInfo.podV6NetworkCIDR != "",
-			"::1",
-			"-6",
+			dstIPv6,
 		},
 	}
-	for _, c := range opensslConnectCommands {
-		if !c.enabled {
+	for _, c := range curlTLSCommands {
```

Comment:
Just `for _, c := range []string{dstIPv4, dstIPv6} {` and remove `curlTLSCommands`?